src/momento/responses/data/list/concatenate_front.py

```diff
@@ -26,8 +27,9 @@ class Success(CacheListConcatenateFrontResponse):
         """The number of values in the list after this concatenation"""
 
     class Error(CacheListConcatenateFrontResponse, ErrorResponseMixin):
-        """Indicates an error occured in the request:
+        """Indicates an error occured in the request.
```

Comment:
```suggestion
        """Indicates an error occurred in the request.
```

Reply:
Looks like this typo is pretty pervasive and could use a batch find and replace.